### ISSUE TITLE
How can I adjust the height of the OSC ?

### ISSUE THREAD

**Issue by htyxyt** [zydezu/ModernX#43]
I tried adjusting the osc_geo.h, but it only reduced the height of area for active mouse input. The display height of the OSC, meaning the semi-transparent part, remained the same as before. 
Could you please tell me which parameter I should modify to adjust the overall height of the OSC?

**Comment by zydezu** [comment]
Are you talking about `OSCfadealpha` ?

**Comment by htyxyt** [comment]
> Are you talking about `OSCfadealpha` ?
No. I tried modifying modern.conf->OSCfadealpha, but it still doesn't seem to work.

What I mean is as shown in the image: when the OSC (On-Screen Controller) is displayed, it gradually becomes transparent from the bottom of the window to the red line. I hope to adjust this height, for example, to make it gradually transparent from the bottom of the window to the green line instead.
![image](https://github.com/user-attachments/assets/3915c37d-ad00-4acb-95cc-16f8a09ad629)

**Comment by zydezu** [comment]
fade_alpha = 100,                       -- alpha of the title bar background box
    fade_blur_strength = 75,                -- blur strength for the OSC alpha fade - caution: high values can take a lot of CPU time to render
    title_bar_fade_alpha = 150,             -- alpha of the OSC background box
    title_bar_fade_blur_strength = 100,     -- blur strength for the title bar alpha fade
    window_fade_alpha = 75,                 -- alpha of the window title bar

these options will probably help you

**Comment by htyxyt** [comment]
> ```
> fade_alpha = 100,                       -- alpha of the title bar background box
> fade_blur_strength = 75,                -- blur strength for the OSC alpha fade - caution: high values can take a lot of CPU time to render
> title_bar_fade_alpha = 150,             -- alpha of the OSC background box
> title_bar_fade_blur_strength = 100,     -- blur strength for the title bar alpha fade
> window_fade_alpha = 75,                 -- alpha of the window title bar
> ```
> 
> these options will probably help you

Thank you very much for your explanation!
